datamodel/app/functions/audit_functions.sql

```diff
@@ -0,0 +1,295 @@
+-----------------------------------------------
+-----------------------------------------------
+-- Provide audit functions for pg_history_viewer
+-----------------------------------------------
+-----------------------------------------------
+
+CREATE OR REPLACE FUNCTION tww_sys.if_modified_func() RETURNS TRIGGER AS $body$
+DECLARE
+    audit_row tww_sys.logged_actions;
```

Comment:
Is it correct to declare a type in that way? What would the correct way to do it?